Vagrantfile

```diff
@@ -104,8 +104,10 @@ Vagrant.configure('2') do |config|
     end
 
     app_gen = <<-'SCRIPT'.gsub(/^\s+/, '')
-      mkdir -p "$HOME/app_service"
+      mkdir -p "$HOME/app_service/lookups"
```

Comment:
I don't think this change is actually doing anything because we do an `rm -rf `on the parent directory immediately afterwards.